nobrainer/models/asdnet.py

```diff
@@ -0,0 +1 @@
+# asdnet tensorflow implementation
```

Comment:
will this be implemented? if not perhaps remove this file

Reply:
I will remove this for now